src/Sentry/Internal/Hub.cs

```diff
@@ -57,7 +57,14 @@ internal Hub(ISentryClient client, ISystemClock clock, ISessionManager sessionMa
                 foreach (var integration in _integrations)
                 {
                     options.DiagnosticLogger?.LogDebug("Registering integration: '{0}'.", integration.GetType().Name);
-                    integration.Register(this, options);
+                    try
+                    {
+                        integration.Register(this, options);
```

Comment:
Do we want to hide integrations that fail installing? Before the user at least would be aware and raise an issue